packages/core/src/2d/sprite/SpriteRenderer.ts

```diff
@@ -284,13 +294,13 @@ export class SpriteRenderer extends Renderer {
    * @internal
    */
   override _updateShaderData(context: RenderContext, onlyMVP: boolean): void {
-    if (onlyMVP) {
+    if (this.getMaterial() === this.engine._spriteDefaultMaterial || onlyMVP) {
```

Comment:
Why add `this.getMaterial() === this.engine._spriteDefaultMaterial` this condition?